main.py

```diff
@@ -144,6 +191,66 @@ def reload_table(self):
             self.tableEmployees.insert('', tk.END, values=(
                 emp, DATA[emp][0], DATA[emp][1], DATA[emp][2], float(DATA[emp][3]), int(DATA[emp][4])))
 
+    def validate_textFieldId(self, event):
+        try:
+            int(self.textFiledId.get())
+            self.textFiledId.config(background="white", foreground="black")
+            return True
+        except ValueError:
+            self.textFiledId.config(background="red", foreground="black")
+            # self.textFiledId.bind("<FocusOut>", self.validate_textFieldId(10))
+            return False
+
+    def validate_textFieldName(self, event):
+        try:
+            if len(self.textFiledName.get()) != 0:
```

Comment:
Please try to decompose the code which is respiting.

Reply:
The code is still respiting. 👨‍💻

Reply:
It was, it is and it will respit.